### PR TITLE
feat(tooltip): manual testing

### PR DESCRIPTION
**Pull request by nathalielindqvist**
Added test for the Tooltip component.

### CODE REVIEW

File: src/pages/FormPage.tsx [hunk 1]
```diff
@@ -358,8 +358,18 @@ const FormPage = () => {
...
+            selector="#anonymously"
+            mouse-over-tooltip="false"
+          >
+            <p className="tds-detail-05 tds-u-m0 tooltip-paragraph">
+              This option is required in order to submit the form.
+            </p>
+          </tds-tooltip>
             <tds-toggle required name="toggle" size="sm">
-              <div slot="label">Answer anonymously</div>
+              <div slot="label" id="anonymously">Answer anonymously
```
Comment (mJarsater) [inline src/pages/FormPage.tsx:371]: I would suggest targeting this with CSS and setting:

display: flex;
align-items: center;
gap: 4px;

and then remove the inline styling on the icon.

**Review by alexandereneroth — CHANGES_REQUESTED** [review]
The 'show' prop needs testing too
https://tegel-storybook.netlify.app/?path=/info/components--tooltip

**Comment by nathalielindqvist** [comment]
> The 'show' prop needs testing too https://tegel-storybook.netlify.app/?path=/info/components--tooltip

Great stuff, updated now! 😄

**Comment by alexandereneroth** [comment]
Hmm seems like the 'show' prop only shows it initially - if you hover in and out of it it closes. 
I don't think thats expected behavior, prolly needs a ticket

**Review by alexandereneroth — APPROVED** [review]

**Comment by nathalielindqvist** [comment]
> Hmm seems like the 'show' prop only shows it initially - if you hover in and out of it it closes. I don't think thats expected behavior, prolly needs a ticket

Yeah I noticed that too. Will write a ticket for it in Jira.

**Comment by mJarsater** [comment]
> > Hmm seems like the 'show' prop only shows it initially - if you hover in and out of it it closes. I don't think thats expected behavior, prolly needs a ticket
> 
> Yeah I noticed that too. Will write a ticket for it in Jira.

I think you could try and changing the value of the show prop based on hover. I think thats how it's inteded to be used.

File: src/pages/FormPage.tsx [hunk 2]
```diff
@@ -358,8 +358,21 @@ const FormPage = () => {
...
+          <tds-tooltip
+            placement="right"
+            selector="#anonymously"
+            mouse-over-tooltip="true"
+            show
+          >
+            <p className="tds-detail-05 tds-u-m0 tooltip-paragraph">
+              This option is required in order to submit the form.
+            </p>
+          </tds-tooltip>
```
Comment (mJarsater) [inline src/pages/FormPage.tsx:370]:
```suggestion
          <tds-tooltip
            placement="right"
            mouse-over-tooltip="true"
            show={showTooltip}
          >
            <p className="tds-detail-05 tds-u-m0 tooltip-paragraph">
              This option is required in order to submit the form.
            </p>
          </tds-tooltip>
```

File: src/pages/FormPage.tsx [hunk 3]
```diff
@@ -358,8 +358,21 @@ const FormPage = () => {
...
             <tds-toggle required name="toggle" size="sm">
-              <div slot="label">Answer anonymously</div>
+              <div slot="label" id="anonymously">
+                Answer anonymously
+                <tds-icon name="info" size="16px"></tds-icon>
```
Comment (mJarsater) [inline src/pages/FormPage.tsx:374]:
```suggestion
                <tds-icon 
                  onMouseEnter={setShowTooltip(true)}
                  onMouseLeave={setShowTooltip(false)}
                  name="info" size="16px"></tds-icon>
```

**Comment by nathalielindqvist** [comment]
'show' prop is not put into action properly in the component, writing ticket in Jira.

**Pull request merged**